Source/Core/DolphinQt/Debugger/RegisterWidget.cpp

```diff
@@ -221,6 +243,61 @@ void RegisterWidget::ShowContextMenu()
       m_updating = false;
     });
 
+    connect(view_hex_column, &QAction::triggered, [this, item] {
+      m_updating = true;
+      auto col = m_table->currentItem()->column();
+      for (int i = 0; i < 32; i++)
+      {
+        auto* update_item = static_cast<RegisterColumn*>(m_table->item(i, col));
+        update_item->SetDisplay(RegisterDisplay::Hex);
+      }
+      m_updating = false;
```

Comment:
This code is practically identical across the 5 connects, put it into a function or something.

Reply:
I was thinking about that, but how do I tell it which RegisterDisplay to use?  Or do I have 5 connects to determine the RegisterDisplay and then one for loop?